problems/arrayMethods.js

```diff
@@ -108,7 +180,15 @@ const doubled = () => {};
  * @returns {boolean} Each number times 3 less than 20 ?
  */
 
-const tripledAndLessThan20 = () => {};
+const tripledAndLessThan20 = (nums) => {
+return nums.every((num) =>{
+  if (num * 3 <= 20){
```

Comment:
We don't want an equal here because the instructions ask for less than 20

This might pass but in scenarios where num * 3 it'll return true even though we want to return false